src-docs/src/views/flex/flex_basis_string.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import React from 'react';
+
+import { OuiFlexGroup, OuiFlexItem } from '../../../../src/components/flex';
+
+export default () => (
+  <div>
+    <OuiFlexGroup>
+      <OuiFlexItem basis={'auto'}>
+        Auto
```

Comment:
Because the actual value of the prop we want users to use is `auto`, we should use `auto` here and not `Auto`. Also, we should perhaps wrap it in `OuiCode`?

The same goes for all of the other values too.

@kgcreative

Reply:
@AMoo-Miki,

Updated text in examples for Basis in Flex Item for **all** basis properties (e.g.`Auto` to `auto`). Just one question what is this suggestion of wrapping with `OuiCode`?